### PR TITLE
New Cheat Sheet: Python Turtle

### PR DESCRIPTION
**Pull request by Tawsif1**
IA page: https://duck.co/ia/view/python_turtle_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@Tawsif1 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/python-turtle.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
+{
+    "id": "python_turtle_functions_cheat_sheet",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/python-turtle.json]: Please could you change the ID to "python_turtle_cheat_sheet" ? That should remove the Travis check error and make it possible to install on the beta server.

File: share/goodie/cheat_sheets/json/python-turtle.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    "name": "Python Turtle Cheat Sheet",
+    "description": "Python Turtle functions",
+    
+    "metadata": {
+        "sourceName": "PythonDocs",
+        "sourceUrl": "https://docs.python.org/2/library/turtle.html"
+    },
+    
+    "aliases": [
+        "python", "turtle", "functions", "methods", "built in", "how to use python turtle"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/python-turtle.json]: I'm afraid these aliases are mostly to general. Please could you change the list to:
"python turtle functions", "python turtle methods"

This should be enough to start with, and we can add more later after it's released if necessary.

File: share/goodie/cheat_sheets/json/text-to-binary.json [hunk 3]
```diff
@@ -0,0 +1,325 @@
+{
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/text-to-binary.json]: Please could you remove this file from your pull request?

**Comment by tagawa** [comment]
Thank you for this cheatsheet @Tawsif1. It looks good but requires a couple of small changes to make it work (I left comments in the file), including removing the `text-to-binary.json` file. Then it should be able to install it on the beta server for testing.

File: share/goodie/cheat_sheets/json/python-turtle.json [hunk 4]
```diff
@@ -0,0 +1,225 @@
+{
+    "id": "python_turtle_functions_cheat_sheet",
+    "name": "Python Turtle Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/python-turtle.json]: Lets change name to "Python Turtle"

File: share/goodie/cheat_sheets/json/python-turtle.json [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+                "val": "Returns the turtle's coordinates",
+                "key": "position()"
+            }, {
+                "val": "Returns the angle of the line from the turtle's position to (x, y)",
+                "key": "towards(x, y)"
+            }, {
+                "val": "Returns the turtle's x coordinate",
+                "key": "xcor()"
+            }, {
+                "val": "returns the turtle's y coordinate",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/python-turtle.json]: Start the val field with a capital letter

**Comment by pjhampton** [comment]
It looks like this PR has gone stale

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**